### PR TITLE
feat: add EC2 instance to metrics module using CDS EC2 module

### PR DESCRIPTION
**Pull request by maxneuvians**
This PR adds a new EC2 instance to the metrics module that can be used to generate report data for distribution. The EC2 instance is added by a custom CDS module.

### CODE REVIEW

File: server/aws/modules/metrics/ec2_data_module.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module "metrics_data_vm" {
+  source          = "github.com/cds-snc/terraform-ec2-module"
```
Comment (CalvinRodo) [inline server/aws/modules/metrics/ec2_data_module.tf:2]: I tagged the module in github so we can work on it without drifting our other environments. 

can you add `?ref=v1.0.0` to the end of this url

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
Just one change.

File: server/aws/modules/metrics/ec2_data_module.tf (hunk 1 above)
Comment (maxneuvians) [reply server/aws/modules/metrics/ec2_data_module.tf]: Pushed

**Review by CalvinRodo — APPROVED** [review]
LGTM

**Pull request merged**